### PR TITLE
128K limit, chattr, rsync, other additions

### PR DESCRIPTION
**Pull request by Lakshmipathi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,8 @@ Notes:
 
 - Learn to use `apt-get`, `yum`, `dnf` or `pacman` (depending on distro) to find and install packages. And make sure you have `pip` to install Python-based command-line tools (a few below are easiest to install via `pip`).
 
+- If you are a newcomer use `Nano` text editor, its the easiest to learn compared to `Vim` and `Emcas`.
```
Comment (tim-teufel) [inline README.md]: its → it's
Emcas → Emacs

Comment (Lakshmipathi) [inline README.md]: will make above correction and resend the request.

**Comment by jlevy** [comment]
Normally we prefer separate PRs for each item, but these are good so let's keep iterating on this one.

Can we remove the nano one altogether for now and let's do it separately? It still should be joined with the vim item, and has some issues (capitalization is wrong). I'd like to reconcile it with #129 after we merge this.

**Comment by jlevy** [comment]
These a great additions —thanks!

**Comment by Lakshmipathi** [comment]
After the first PR created, I was unable to find 'Create new pull request' for the second item and after that.(may be I missed something).

**Comment by tim-teufel** [comment]
You merge branches, not individual commits. Just create so-called "feature branches" for each separate item for the future.

**Comment by jlevy** [comment]
@Lakshmipathi do you mind deleting the nano item for now? I'll handle that one myself.

But I'm happy with the rest of the items here.

I'll still leave this open for a day or two in case anyone else has feedback/suggestions.

**Comment by Lakshmipathi** [comment]
@tim-teufel thanks for the details. I never sent a multiple pull requests at one instance. 
@jlevy ,  Sorry I didn't understand about deleting the nano item. Is it option with Github or should I remove/delete that change with another patch?

**Comment by jlevy** [comment]
You can just add another commit to remove the nano item. Will add it back later. Thanks!

**Comment by Lakshmipathi** [comment]
okay, removed the nano tip and committed the changes.

**Comment by jlevy** [comment]
Thank you @Lakshmipathi !

**Comment by jlevy** [comment]
And @tim-teufel for the assist.

**Comment by Lakshmipathi** [comment]
@tim-teufel , @jlevy  thanks for the review comments.

**Pull request merged**